importers/aircraft_csv_format_importer.py

```diff
@@ -0,0 +1,155 @@
+from datetime import datetime
+
+from pepys_import.core.formats import unit_registry
+from pepys_import.core.formats.location import Location
+from pepys_import.core.validators import constants
+from pepys_import.file.highlighter.support.combine import combine_tokens
+from pepys_import.file.importer import Importer
+from pepys_import.utils.sqlalchemy_utils import get_lowest_privacy
+from pepys_import.utils.unit_utils import convert_absolute_angle, convert_distance, convert_speed
+
+
+class AircraftCsvFormatImporter(Importer):
+    def __init__(self):
+        super().__init__(
+            name="Aircraft CSV Format Importer",
+            validation_level=constants.BASIC_LEVEL,
+            short_name=" Aircraft CSV Importer",
+            default_privacy="Public",
+            datafile_type="CSV",
+        )
+        self.text_label = None
+
+    def can_load_this_type(self, suffix):
+        return suffix.upper() == ".CSV"
+
+    def can_load_this_filename(self, filename):
+        return True
+
+    def can_load_this_header(self, header):
+        return header.startswith("Date(Uk)")
+
+    def can_load_this_file(self, file_contents):
+        return True
+
+    def _load_this_line(self, data_store, line_number, line, datafile, change_id):
+        # Skip the header
+        if line_number == 1:
+            return
+
+        tokens = line.tokens(line.CSV_TOKENISER, ",")
+        if len(tokens) <= 1:
+            # The last line may be empty, don't worry
+            return
+        if len(tokens) < 8:
```

Comment:
This tests for too few tokens, but what if there are too many? Should we be checking for exactly 9?

(Some of the other importers don't check for an exact number of tokens, as the lines in some formats can have a free-text comment at the end, which means the total number of tokens can be very high - but that doesn't apply here)